### DISCUSSION TITLE
디자인 시스템 / 타입 관련 참고 자료 입니다.

### DISCUSSION THREAD

**Discussion by godhyeongman** [eRentronic/eRentronic-Client#52] [일반]
### 디자인 시스템 레포 : https://github.com/cobaltinc/co-design
저희랑 컨벤션 같은건 다르니깐 https://www.cobalt.run/16459edc-27a6-4226-992b-bdec45e379be 이 영상 보고 참고 정도만 하면 좋을것 같아요 :)

### 타입 관리 관련 : https://github.com/woowacourse-teams/2021-pick-git/tree/develop/frontend/src
우테코 프로젝트 레포인데 저희 논의 했던 것 처럼 타입 폴더를 두고 이를 개발 하더라구요 참고 가능할것 같습니다.

### 다향성 컴포넌트 유데미 자료
1. https://www.udemy.com/course/learning-path-react-master-design-patterns-with-react/
2. https://kciter.so/posts/polymorphic-react-component

**Comment by herrakam** [comment]
이부분도 정리를 해주셨군요 감사합니다!!
